pkg/virt-api/webhooks/validating-webhook/validating-webhook_test.go

```diff
@@ -76,6 +76,33 @@ var _ = Describe("Validating Webhook", func() {
 			Expect(len(resp.Result.Details.Causes)).To(Equal(1))
 			Expect(resp.Result.Details.Causes[0].Field).To(Equal("spec.domain.devices.disks[0].volumeName"))
 		})
+		It("should reject VMIs witht memory set if after presets are applied", func() {
```

Comment:
TYPO and re-phrase I do not think that specify preset here is really important

Reply:
Fixed